docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -320,6 +320,132 @@ will be lost.
 $ kubectl delete -f alluxio-journal-volume.yaml
 ```
 
+#### Upgrade Alluxio
+
+This section will go over how to upgrade Alluxio in your Kubernetes cluster with `kubectl`.
+This section will only cover the basic procedure of how to kill all existing Pods, upgrade the version and bring the Alluxio master and worker Pods back up.
+More advanced rolling update procedure will be covered in the next section. 
+
+**Step 1: Upgrade the docker image version tag**
+ 
+Each released Alluxio version will have the corresponding docker image released on [dockerhub](https://hub.docker.com/r/alluxio/alluxio).
+
+You should update the `image` field of all the Alluxio containers to use the target version tag. Tag `latest` will point to the latest stable version.
+It is recommended that Alluxio masters and workers are running on the same version for the best compatibility.
+
+For example, if you want to upgrade Alluxio to the latest stable version, update the containers as below:
+
+```yaml
+containers:
+- name: alluxio-master
+  image: alluxio/alluxio:latest
+  imagePullPolicy: IfNotPresent
+  ...
+- name: alluxio-job-master
+  image: alluxio/alluxio:latest
+  imagePullPolicy: IfNotPresent
+  ...
+```
+
+**Step 2: Format journal if necessary**
+
+Note that whether you need to re-format the journal and lose all file metadata depends on whether the journals are compatible.
+If you are upgrading from v1.8 to v2.x, you have to format the journal. And by doing that you lose all the journal and file metadata.
+If you are upgrading from v2.x to a higher minor version v2.y, or from v2.x.y to a higher minor version v2.x.z,
+the higher version Alluxio master will be able to read the lower version Alluxio master journal to recover all the journal and file metadata.
+
+If you need to format the journal, you can update the YAML file for Alluxio master containers to specify this on master startup time.
+Remove the `--no-format` option from Alluxio master startup arguments to force the re-format of journal as illustrated below.
```

Comment:
we should not suggest this way. every master restart will lead to a format, instead an init container should be used to force format

Reply:
In the updated doc I am pointing to a `alluxio-format-master.yaml` Job that will be added in a separate PR. I'm not including that in this PR because there's more than one thing to do to include that `alluxio-format-master.yaml`, I need to helm-template it and possible add something for formatting workers too.